.github/workflows/bow-php-action.yml

```diff
@@ -15,27 +11,28 @@ env:
 jobs:
   build-and-test:
     runs-on: ${{ matrix.os }}
-    name: PHP-${{ matrix.php }} - ${{ matrix.os }}
-
-    services:
-      mysql:
-        image: mysql:5.7
-        env:
-          MYSQL_DATABASE: test_db
-          MYSQL_ALLOW_EMPTY_PASSWORD: yes
-        ports:
-          - 3306
-        options: --health-cmd="mysqladmin ping" --health-interval=10s --health-timeout=5s --health-retries=3
-
     strategy:
       matrix:
         php: [7.4, 7.3, 7.2]
         os: [ubuntu-latest]
 
+    name: PHP-${{ matrix.php }} - ${{ matrix.os }}
+
     steps:
       - name: Checkout code
         uses: actions/checkout@v2
 
+      - name: Setup MySQL
+        uses: mirromutth/mysql-action@v1.1
```

Comment:
Using the `mirromutth/mysql-action` to replace the default GitHub action MySQL service.